PAY-621: Voids summary (#1080)
magento/merchdocs: src/magento-payments/voids.md

---
title: Voids
---

> With Magento Payments, you can utilize existing Magento functionality for voiding transactions. Voids allow you to stop payment on a specific order, if the funds are not yet captured. After you void an order, you can cancel orders in `Pending` status and issue any needed refunds.

Reviewer: Definitely need Craig's $0.02 here but I think it might be more correct to say "Voids allow you to release a hold on funds for a specific order".

Reviewer 2: Voids allow you to stop payment on a specific order, if the funds are not yet captured. After you void an order, you can cancel orders in Pending status and issue any needed refunds.
to
Voids allow you to liberate funds blocked by authorisation for an order as long as it has not yet been captured. After you void an authorisation, you can cancel the order in Pending status. Alternatively, you could just choose to cancel an order and any uncaptured authorisations will also be voided.

(extra note of explanation - refunds are only possible/necessary on captured transactions).

---
title: Voids
---

With Magento Payments, you can utilize existing Magento functionality for voiding transactions. Voids allow you to free the funds in a credit or debit card account that are blocked or held aside by an authorization for the amount of a purchase.

{:.bs-callout-info}
You can only void a transaction if the funds are not yet captured *link to authorize and capture payment method*.

When a customer purchases an item from your store, if you have configured your store to authorize only (not capture) at the point of sale *LINK TO FUTURE REF DOCS at docs.magento.com/user-guide/configuration/sales/magento-payments*, the order has a `Processing` status in the Magento Admin.

When that order is still processing, you can void the transaction. After you void it, you can cancel the order, which is in a `Pending` status, and issue any needed refunds via [credit memo]({% link sales/credit-memos.md %}).

> You can also [cancel an order]({% link sales/order-update.md %}#cancel-a-pending-order) that has been settled (such as when it has a `Complete` or `Closed` status) or for which payment was captured, and any un-captured authorizations will also be voided as part of that cancellation process.

Reviewer 3: A little more simplified and without a third verb tense. 

```suggestion
You can also [cancel an order]({% link sales/order-update.md %}#cancel-a-pending-order) that has been settled (such as when it has a `Complete` or `Closed` status) or for which payment was captured. Any un-captured authorizations are also voided as part of that cancellation process.
```
